robot_specific_config/abb_irb2600_support/urdf/irb2600.ros2_control.xacro

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="abb_ros2_control" params="name prefix use_fake_hardware:=true rws_ip rws_port">
+    <ros2_control name="${name}" type="system">
+      <hardware>
+        <!-- ros2_control simulation -->
+        <xacro:if value="${use_fake_hardware}">
+          <plugin>mock_components/GenericSystem</plugin>
+          <param name="fake_sensor_commands">false</param>
+          <param name="state_following_offset">0.0</param>
+        </xacro:if>
+        <!-- physical hardware or RobotStudio simulation -->
+        <xacro:unless value="${use_fake_hardware}">
+          <plugin>abb_hardware_interface/ABBSystemHardware</plugin>
+          <param name="rws_port">${rws_port}</param>
+          <param name="rws_ip">${rws_ip}</param>
+          <!-- The following parameter is used for non-MultiMove only -->
+          <param name="egm_port">6511</param>
+          <!-- The following parameters are used for the MultiMove example only -->
+          <!-- <param name="rob1egm_port">6511</param> -->
+          <!-- <param name="extaxegm_port">6512</param> -->
+        </xacro:unless>
+      </hardware>
+      <joint name="${prefix}joint_1">
+        <command_interface name="position">
+          <param name="min">{-2*pi}</param>
```

Comment:
None of the joint command interface limits here match the limits specified in the urdf. Kindly update these values.

Reply:
fixed in commit [e633967](https://github.com/PickNikRobotics/abb_ros2/pull/53/commits/e6339671fa35b2cd25fc848a7da47aac8ea0ae33)